src/enter.js

```diff
@@ -26,6 +26,6 @@ export default class Enter extends Feature {
 		this.listenTo( editingView, 'enter', ( evt, data ) => {
 			editor.execute( 'enter' );
 			data.preventDefault();
-		} );
+		}, 'low' );
```

Comment:
> And there is low added after this listener

What do you mean?

> I am not sure now how this chain of dependencies and requirements in plugins will work

Works like a charm :P. No other feature requires the enter feature right now, so it can be loaded first or last.